src/ArchMap.cpp

```diff
@@ -149,7 +149,22 @@ std::string SleighIdFromCore(RCore *core)
 {
 	const char *arch = r_config_get(core->config, "asm.arch");
 	auto arch_it = arch_map.find(arch);
```

Comment:
maybe if (!strcmp (arch, "r2ghidra")) ?

Reply:
Yes, better check this right away and use the same sleigh id as the asm/anal plugins instead of just falling back to it.